modules/apps/object/object-web/src/main/resources/META-INF/resources/js/utils/fdsSourceDataRenderer.tsx

```diff
@@ -0,0 +1,30 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import classNames from 'classnames';
+import React from 'react';
+
+interface fdsSourceDataRenderer {
```

Comment:
@pedro-oliveira446, please rename this interface name to `FDSSourceDataRendererProps`